README.md

```diff
@@ -6,6 +6,95 @@ storage of a smaller artefact for tasks such as regression tests, while omitting
 metadata and data attributes that may change between test executions (such as
 timestamps in history attributes).
 
+## Features
+
+### Generating hashed files
```

Comment:
Ooops. I'll update the README now.

Reply:
Done: [8a5672d](https://github.com/nasa/earthdata-hashdiff/pull/3/commits/8a5672d17286949f3ab08d186bb3b802489f5379)